### ISSUE TITLE
ETHOS 1.5 and G-RXx Receivers should startup at zero or last altitude 

### ISSUE THREAD

**Issue by spoke2570** [FrSkyRC/ETHOS-Feedback-Community#3538]
Currently the  when a model is loaded the FrSky built in altimeter jumps to some value that is not the last registered when the model was unloaded or the radio turned off. The few receivers I have all give quite different values at power up. The range of start altitude for my inventory is like 50 to  170 meters. It would be nice for this to be zero meters.

Thanks!

**Comment by kramgrand** [comment]
The GR receivers e.g. GR8 I use have done this since I started using them.  
My automatic work around is to use a logic switch to detect the rise of receiver battery voltage via the telemetry and then to initiate a Special Function reset for a few seconds to the receiver altimeter.  Alternatively, it can be manually reset with a switch.  I agree it would be better for the receiver to register 0m at power on. I've just got used to it I guess.
